lib/_http_client.js

```diff
@@ -129,8 +129,12 @@ function ClientRequest(options, cb) {
     self._storeHeader(self.method + ' ' + self.path + ' HTTP/1.1\r\n',
                       options.headers);
   } else if (self.getHeader('expect')) {
+    if (self._header) {
+      throw new Error('Can\'t render headers after they are sent to the ' +
```

Comment:
It shouldn't, I just had to move the check from the old `_renderHeaders()` to the appropriate call sites since that function no longer exists. I tried just moving it to `_storeHeader()` but that caused problems IIRC.